config/post_update/40005_update_footer_data/oe_corporate_blocks.eu_data.footer.yml

```diff
@@ -0,0 +1,90 @@
+langcode: en
```

Comment:
Did you change anything in the EU footer config/install? Why do you again import this in the update path?

Reply:
Right, it's not needed.